OneHotEncoder.__init__() got an unexpected keyword argument 'sparse_output'
Traceback (most recent call last):
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/script/worker.py", line 127, in work_loop
    run_command(Context(self, run))
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/task/run.py", line 24, in __call__
    self.experiment(context, self.config)
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/task/experiment/pruning_experiment/iterative_pruning_experiment.py", line 67, in __call__
    dataset, metrics, loss_metric = self._load_and_prepare_dataset()
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/task/experiment/training_experiment/training_experiment.py", line 185, in _load_and_prepare_dataset
    dataset = PreparedDataset(
              ^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/prepared_dataset.py", line 17, in __init__
    dataset: Dataset = load_dataset(spec.source, spec.name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_util.py", line 29, in load_dataset
    result = __source_loaders(source)(name)()  # type: ignore
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 55, in __call__
    data = self._prepare_dataset_data(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 125, in _prepare_dataset_data
    data.train = self._prepare_data_group(data.train)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 137, in _prepare_data_group
    group.outputs = self._prepare_outputs(group.outputs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 144, in _prepare_outputs
    return self.prepare_data(data, self.dynamic_output_value_transform)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 151, in prepare_data
    return self.prepare_matrix(value, transform)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 204, in prepare_matrix
    transformed_value = self.prepare_value(value, value_transform)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 194, in prepare_value
    return value_transform(self, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 184, in dynamic_output_value_transform
    return self.one_hot(value)
           ^^^^^^^^^^^^^^^^^^^
  File "/lustre/eaglefs/projects/modularai/dmp/dmp/src/dmp/dataset/dataset_loader.py", line 223, in one_hot
    preprocessor = OneHotEncoder(handle_unknown="ignore", sparse_output=False)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: OneHotEncoder.__init__() got an unexpected keyword argument 'sparse_output'
